File: Assets/common/script/scenery/FieldBackground.js
#pragma strict
var target : Transform;
var rotation : Vector3;
private var rot : Quaternion;

function Start () {
	rot = Quaternion.Euler(rotation.x, rotation.y, rotation.z);
}

function Update () {
	transform.position = target.position;
	transform.rotation *= rot;
}

File: Assets/common/script/scenery/FieldBackground.cs
using UnityEngine;
using System;


public class FieldBackground:MonoBehaviour{
	public Transform target;
	public Vector3 rotation;
	Quaternion rot;
	
	public void Start() {
		rot = Quaternion.Euler(rotation.x, rotation.y, rotation.z);
	}
	
	public void Update() {
		transform.position = target.position;
		transform.rotation *= rot;
	}
}